src/js/dataMapping/search/tableSearchFields.js

```diff
@@ -176,11 +151,9 @@ const tableSearchFields = {
             'recipient_name',
```

Comment:
Can you update the loan tab to use Geoff's columns:

* Award ID
* Recipient Name
* Issued Date
* Loan Value
* Subsidy Cost
* Funding Agency

If this is going to take too long, branch off this branch and do a separate PR with these changes.